README.md

```diff
@@ -45,20 +58,48 @@ Password: Dynamic33
 
 5. **Run the Analysis Code**
 
-In RStudio, open `vancouver-tree-height-geo.qmd` and run all code chunks.
+open a terminal (from the docker rstudio) and run the following commands:
+
+```
+Rscript scripts/00_download_data.R "https://opendata.vancouver.ca/api/explore\
+/v2.1/catalog/datasets/street-trees/exports/csv?lang=en&timezone=\
+America%2FLos_Angeles&use_labels=true&delimiter=%3B" "data/street-trees.csv"
```

Comment:
Actually, the backslashes are really meant to go between arguments, not to be used as line breaks to make things prettier. So the way they are used here is incorrect. For example, if we want to use backslashes, this code chunk should be:

```bash
Rscript scripts/00_download_data.R "https://opendata.vancouver.ca/api/explore/v2.1/catalog/datasets/street-trees/exports/csv?lang=en&timezone=America%2FLos_Angeles&use_labels=true&delimiter=%3B" \
    "data/street-trees.csv"
```